gshoppingflux/gshoppingflux.php

```diff
@@ -1634,7 +1634,7 @@ public function renderInfo()
 		));
 	}
 
-	public function customGetNestedCategories($shop_id, $root_category = null, $id_lang = false, $active = true, $groups = null, $use_shop_restriction = true, $sql_filter = '', $sql_sort = '', $sql_limit = '')
```

Comment:
... and you did well. What about modifying your pull request, just removing the semicolon, by the way I could merge it to the main code repo? I'm pretty sure many other people should have encounter the same issue, you could help everyone doing this :)